### ISSUE TITLE
This source is as good as shit with locked bootloader.

### ISSUE THREAD

**Issue by Crescendo-BLYAT** [MiCode/MiBox_Kernel_OpenSource#7]
How to use this source if you keep the bootloader locked no matter what???

**Comment by exkc** [comment]
@Crescendo-BLYAT  Yes you are right the bootloader(bl) is locked so you cant boot any custom os directly via the stock bl . However you can chain Boot another  bl fom the android 8 's stock bl which mean you can run custom os  on mi 3.

That is how pm os boot on mi box 3 btw.
https://wiki.postmarketos.org/wiki/Xiaomi_Mi_box_3_(xiaomi-once)

Tbh this kernel is useless for the pm os port bc i ported pm os to it without this kernel

**Comment by chkndrp** [comment]
They have to release the source code no matter if the device is unlock-able, or not because of Linux's GPLv2 license
(manufacturers must publish the changes they made to the Linux kernel)
